x/strdburner/keeper/abci.go

```diff
@@ -0,0 +1,20 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/Stride-Labs/stride/v24/x/strdburner/types"
+)
+
+func EndBlocker(ctx sdk.Context, k Keeper) {
+	strdBurnerAddress := k.GetStrdBurnerAddress()
+
+	strdBalance := k.bankKeeper.GetBalance(ctx, strdBurnerAddress, "ustrd")
+
+	if strdBalance.IsPositive() {
+		err := k.bankKeeper.BurnCoins(ctx, types.ModuleName, sdk.NewCoins(strdBalance))
+		if err != nil {
+			k.Logger(ctx).Error("unable to burn %s", strdBalance.String())
+		}
+	}
```

Comment:
might make sense to also log when a burn was successful - shouldn't be too noisy since this will only happen once every 6 hours